app/facades/explore_facade.rb

```diff
@@ -0,0 +1,16 @@
+class ExploreFacade
+  attr_reader :scores
+
+  def initialize(current_user)
+    @users =  User.where.not(id: current_user.id).where.not(flat_id: nil) #Need to make this pull location sorted list.
+    @scores = get_scores
+  end
+
+  def get_scores
+    all_scores = []
+    @users.each do |user|
+      all_scores << Score.create(user.flat_id)
+    end
+    all_scores.flatten
+  end
+end
```

Comment:
This is going to make seperate API calls for each user. I think that's probably unavoidable, but we may want to explore an alternative. This is working, though.